src/docfx/lib/log/Telemetry.cs

```diff
@@ -64,9 +64,9 @@ public static IDisposable TrackingOperationTime(TelemetryName name)
             return new PerfScope(name.ToString());
         }
 
-        public static void TrackErrorCount(string code, ErrorLevel level)
+        public static void TrackErrorCount(string code, string message, ErrorLevel level)
```

Comment:
This may produce too much telemetry. Let me take care of populating attribute/tag name.

```suggestion
        public static void TrackErrorCount(string code, ErrorLevel level)
```

Reply:
the meesage will be like `attribute 'name' on   tag 'a'` or `tag 'token'`. In my last log analysis, there are 140 different values.